bokeh/charts/builders/chord_builder.py

```diff
@@ -0,0 +1,182 @@
+"""This is the Bokeh charts interface. It gives you a high level API
+to build complex plot is a simple way.
+
+This is the Scatter class which lets you build your Scatter charts
+just passing the arguments to the Chart class and calling the proper
+functions.
+"""
+# -----------------------------------------------------------------------------
+# Copyright (c) 2012 - 2015, Continuum Analytics, Inc. All rights reserved.
+#
+# Powered by the Bokeh Development Team.
+#
+# The full license is in the file LICENSE.txt, distributed with this software.
+# -----------------------------------------------------------------------------
+
+# -----------------------------------------------------------------------------
+# Imports
+# -----------------------------------------------------------------------------
+from __future__ import absolute_import
+
+import numpy as np
+import pandas as pd
+from math import cos, sin, pi
+from bokeh.charts.properties import Dimension
+from bokeh.charts.builder import create_and_build, Builder
+from bokeh.charts.attributes import MarkerAttr, ColorAttr, CatAttr
+from bokeh.charts.utils import color_in_equal_space
+from bokeh.models import Range1d
+from bokeh.models.glyphs import Arc, Bezier
+from bokeh.models.renderers import GlyphRenderer
+from bokeh.models.sources import ColumnDataSource
+from bokeh.core.properties import Instance
+
+
+# -----------------------------------------------------------------------------
+# Classes and functions
+# -----------------------------------------------------------------------------
+
+
+def Chord(data=None, agg=None, **kws):
+    """ Create a scatter chart using :class:`ChordBuilder <bokeh.charts.builders.chord_builder.ChordBuilder>`
+    to render the geometry from values.
+
+    Args:
+        data (:ref:`userguide_charts_data_types`):  correlation table
+        labels (str or list(str), optional): the column  of label to use for the x and y
+
+
+    Returns:
+        :class:`Chord`:
+
+    Examples:
+
+    .. bokeh-plot::
+        :source-position: above
+
```

Comment:
Example code is missing here... we could use this https://gist.github.com/merqurio/ab61bb4c7d08cccf4e58, right? Or maybe a simplified version?